cob_bringup/drivers/canopen_402.launch

```diff
@@ -18,7 +18,6 @@
 
 	<!-- start canopen node -->
 	<node ns="$(arg component_name)" pkg="canopen_motor_node" type="canopen_motor_node" name="driver" output="screen">
-		<rosparam command="load" file="$(arg pkg_hardware_config)/robots/$(arg robot)/config/$(arg component_name)_driver.yaml"/>
```

Comment:
I guess removing the parts uploading parameters will break the driver for the robots...